README.rst

```diff
@@ -251,31 +251,63 @@ Log watcher
 
 Configuration variables can be set via Env variables:
 
-- ``WATCHER_CONTAINERS_PATH``: Containers directory path mounted from the host (Default: ``/var/lib/docker/containers``)
-- ``WATCHER_AGENTS``: Comma separated string of required log processor agents. (Required. Example: "scalyr,appdynamics")
-- ``WATCHER_CLUSTER_ID``: Kubernetes Cluster ID.
-- ``WATCHER_KUBE_URL``: URL to API proxy service. Service is expected to handle authentication to the Kubernetes cluster. If set, then log-watcher will not use serviceaccount config.
-- ``WATCHER_KUBERNETES_UPDATE_CERTIFICATES``: [Deprecated] Call update-ca-certificates for Kubernetes service account ca.crt.
-- ``WATCHER_INTERVAL``: Polling interval (secs) for the watcher to detect containers changes. (Default: 60 sec)
-- ``WATCHER_DEBUG``: Verbose output. (Default: False)
+``WATCHER_CONTAINERS_PATH``
+  Containers directory path mounted from the host (Default: ``/var/lib/docker/containers``)
+
+``WATCHER_KUBERNETES_STRICT_LABELS``
+  If set then only containers running in pods with ``application`` and ``version`` metadata labels will be considered for log watching. (Default is ``False``)
+
+  If not set then kubernetes-log-watcher will set ``application_id`` from *pod name*; in order to provide consistent attributes to log configuration agents.
+
+``WATCHER_AGENTS``
+   Comma separated string of required log processor agents. (Required. Example: "scalyr,appdynamics")
```

Comment:
I know this is not new in this PR, just adding feedback now... What are log processors agents? Where could I get more information about them?